torch/csrc/jit/interpreter.cpp

```diff
@@ -181,27 +443,38 @@ Operation createEvalOperation(CppOp * op) {
       output_edges.emplace_back(handle_in->forward_inputs, i);
     auto values = engine.execute(handle_in->forward_outputs, v_inputs, true, create_graph, output_edges);
     for(auto & v : values)
-      outputs.push_back(builder.addOutput(v));
-    builder.writeTo(outputs);
+      stack.push_back(builder.addOutput(v));
+    builder.writeTo(stack);
+    return 0;
   };
 }
 
-using tensor_list = std::vector<at::Tensor>;
 
-tensor_list unsafeToTensorListShare(const list_of_retainable & inputs) {
-  tensor_list tinputs;
-  tinputs.reserve(inputs.size());
-  for(auto & i : inputs) {
-    tinputs.push_back(unsafeToTensorShare(i));
+Operation createConditionalJump(int offset, bool jump_when_cond_is) {
+  if(jump_when_cond_is/*=true*/) {
+    return [offset](Stack & stack) {
+      auto t = at::Scalar(pop(stack)).toLong();
```

Comment:
This Scalar constructor fails when it gets a nelements=1 tensor, for ex from the Python frontend. I got around this by copying over and using `tensor_as` from https://github.com/pytorch/pytorch/pull/5298